pl_bolts/models/detection/__init__.py

```diff
@@ -1,7 +1,5 @@
 from pl_bolts.models.detection import components  # noqa: F401
 from pl_bolts.models.detection.faster_rcnn import FasterRCNN  # noqa: F401
+from pl_bolts.models.detection.retinanet import RetinaNet  # noqa: F401
```

Comment:
We don't need these ignores?
```suggestion
from pl_bolts.models.detection import components
from pl_bolts.models.detection.faster_rcnn import FasterRCNN
from pl_bolts.models.detection.retinanet import RetinaNet
```